optaplanner-core/src/test/java/org/optaplanner/core/impl/score/buildin/bendablebigdecimal/BendableBigDecimalScoreDefinitionTest.java

```diff
@@ -21,10 +21,24 @@
 import org.junit.Test;
 import org.optaplanner.core.api.score.buildin.bendablebigdecimal.BendableBigDecimalScore;
 
-import static org.junit.Assert.*;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.Assert.assertArrayEquals;
+import static org.junit.Assert.assertEquals;
 
 public class BendableBigDecimalScoreDefinitionTest {
 
+    @Test
+    public void getZeroScore() {
+        BendableBigDecimalScore score = new BendableBigDecimalScoreDefinition(1, 2).getZeroScore();
+        assertThat(score.toLevelNumbers()).containsOnly(BigDecimal.ZERO);
```

Comment:
I recommend avoid toLevelNumbers if possible.
So I'd assert that getZeroScore equals `BendableScore.of(new int[]{0}, new int[]{0, 0})`